src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php

```diff
@@ -37,7 +37,7 @@ public function __construct(ValidatorInterface $validator)
 
         /* @var $metadata ClassMetadata */
         $metadata->addConstraint(new Form());
-        $metadata->addPropertyConstraint('children', new Valid());
+        $metadata->traversalStrategy = TraversalStrategy::NONE;
```

Comment:
Based on the comment preceding this, don't we need to do a similar change in the XML file ?

Reply:
Indeed, that missing sync was also the root cause for the issue reported by @lyrixx in https://github.com/symfony/symfony/issues/36852#issuecomment-634635009.